src/Admin/Routing/RedirectRouteAdmin.php

```diff
@@ -38,12 +38,23 @@ protected function configureListFields(ListMapper $listMapper)
     protected function configureFormFields(FormMapper $formMapper)
     {
         $formMapper
-            ->with('form.group_general')
-                ->add('parentDocument', TreeModelType::class, array('choice_list' => array(), 'select_root_node' => true, 'root_node' => $this->routeRoot))
+            ->with('form.group_location', ['class' => 'col-md-3'])
```

Comment:
Not needed, if the field has no translation domain, the domain of the admin will be used.